src/router/routes.ts

```diff
@@ -151,7 +162,7 @@ export default (store: Store<State>): RouteRecordRaw[] => {
               name: 'gateway-detail-view',
               meta: {
                 title: 'Gateway',
-                parent: 'gateway-list-view',
+                isBreadcrumb: true,
                 breadcrumbTitleParam: 'dataPlane',
```

Comment:
I know its not part of this PR and more part of the ongoing breadcrumb chatter, but these 'specify the data property param to use for the breadcrumb text' type things would go away if we just did this in the template. (just to be clear no action requested here)